cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -169,17 +168,20 @@ public static ConsentLibBuilder newBuilder(Integer accountId, String property, I
         shouldCleanConsentOnError = b.shouldCleanConsentOnError;
         onBeforeSendingConsent = b.onBeforeSendingConsent;
         onNoIntentActivitiesFound = b.onNoIntentActivitiesFound;
-        this.isOTT = b.isOTT;
-
-        //TODO: inject consoleWebview from the builder as well (overload/callbacks refactor required)
-        webView = buildWebView(b.getContext());
+        isOTT = b.isOTT;
 
+        uiThreadHandler = b.getUIThreadHandler();
         mCountDownTimer = b.getTimer(onCountdownFinished());
-
         sourcePoint = b.getSourcePointClient();
-
         storeClient = b.getStoreClient();
+
+        // :warning: following methods depend on storeClient initialization
         setConsentData(b.authId);
+        try {
+            webView = buildWebView(b.getContext());
```

Comment:
@leodoin what if we used the fix you implemented but moved the instantiation to the webview to a separate method? Something like:
```java
ConsentWebView getWebView() {
  return webview == null ? buildWebView(context) : webview;
}
```

The downside of this approach is that we'd have to keep a reference to the `context`. But that object is already in memory and it's not going to be released throughout the whole app lifecycle. Meanwhile the `webview` is not used the majority of the time (users already gave consent). 

What do you think?